kvx/analysis.sed

```diff
@@ -5,9 +5,9 @@
 /"errop/	{s/.type =/& RZ_ANALYSIS_OP_TYPE_TRAP |/};
 /"abs/		{s/.type =/& RZ_ANALYSIS_OP_TYPE_ABS |/};
 /"copy/		{s/.type =/& RZ_ANALYSIS_OP_TYPE_MOV |/};
-/"xcopy/	{s/.type =/& RZ_ANALYSIS_OP_TYPE_MOV |/};
```

Comment:
this and 2 other instructions has been removed. is this correct?

Reply:
yes, such instructions (xcopy, xso, xlo) are aliases and are no longer present in binutils sources